soc/posix/inf_clock/Kconfig

```
@@ -10,3 +10,31 @@ config NATIVE_SIMULATOR_CPU_N
 	  Which native simulator embedded CPU number is this image targeting.
 	  This option is only applicable for targets which use the
 	  native simulator as their runner.
+
+config NATIVE_SIMULATOR_N_MCUS
+	int "Total number of MCUs this target has"
+	range 1 16
+	default 1
+	depends on NATIVE_LIBRARY
+	help
+	  How many AMP MCUs does this target have in total.
+
+config NATIVE_SIMULATOR_PRIMARY_MCU_N
```

I guess this is based on some index, right? Maybe
```suggestion
config NATIVE_SIMULATOR_PRIMARY_MCU_INDEX
```
or
```suggestion
config NATIVE_SIMULATOR_PRIMARY_MCU_ID
```

---

Do you feel strongly enough about this like to block the PR? These 2 options are not really for users but boards definitions.
(there is 20 commits on top of these ones.. so these rename are quite painful.)

---

Let's get another set of eyes on this, and if someone else agrees that they are fine, we'll keep them as is :) 

@jori-nordic What do you think?

---

I think they're fine. Maybe just in case you could make them promptless @aescolar ? That way we can rename later if really necessary.

---

If I make them promptless it is not possible anymore to set them in defconfigs or project .confs
NATIVE_SIMULATOR_PRIMARY_MCU_N I expect users could want to change

---

Alright, @aescolar please then just update the other comments and leave these as is

---

I did the changes you requested... they were really painful..